monkestation/code/game/objects/structures/signs/_signs.dm

```diff
@@ -0,0 +1,20 @@
+/obj/structure/sign
+	//Scaling Posters
+	var/size = 1
+
+/obj/structure/sign/attack_hand(mob/user)
```

Comment:
Also, A concern I have is that you're unable to rip posters and such, make sure that works

Reply:
You are able to rip posters. You're commenting on /obj/structure/sign rather than /obj/structure/sign/poster
It requires harm intent to tear a poster down, but help intent to tap the sign.

Reply:
I cant really look at the full proc, just making sure you are calling the parent